lib/rules/verify-constants.ts

```diff
@@ -173,10 +172,42 @@ const rule: Rule.RuleModule = {
                   });
                 }
               } else if (!options?.ignoreUnrecognized && segment !== "") {
-                context.report({
-                  node,
-                  message: `Unrecognized enumerated value name "${segment}"`,
-                });
+                const options: string[] = [];
+                for (const data of tagInfo.data) {
+                  if (data.toLowerCase().includes(lowerCaseSegment)) {
+                    options.push(data);
+                  }
+                }
+                if (options.length > 1) {
+                  const suggestions = options.map(
+                    (opt): Rule.SuggestionReportDescriptor => {
+                      return {
+                        desc: `Change enumerated value to ${opt}`,
+                        fix: (fixer: Rule.RuleFixer) => {
+                          return fixer.replaceTextRange(range, opt);
+                        },
+                      };
+                    }
```

Comment:
```suggestion
                    opt => ({
                      desc: `Change enumerated value to ${opt}`,
                      fix: fixer => {
                        return fixer.replaceTextRange(range, opt);
                      },
                    })
```
I don't think you need the type annotations here?

Reply:
Ah, yeah, I threw that in there to figure out how to construct the data. I'll remove that.

Reply:
413d35512684c9876e216ef333507f370ff036a8 does as much of this suggestion as possible